src/main/java/com/mgmtp/radio/domain/youtube/YouTubeVideo.java

```diff
@@ -0,0 +1,47 @@
+package com.mgmtp.radio.domain.youtube;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "youtube")
+@Data
+@NoArgsConstructor
+public class YouTubeVideo {
+    private String title;
+    private String url;
+    private String thumbnailUrl;
+    private String description;
+
+    public String getTitle() {
```

Comment:
No need set, get because we are using Lombok. We use @Data, that will be generate the set, get methods

Reply:
This is a redundant file. I removed it.